CharacterCreation/02_Classes.txt

```diff
@@ -44,40 +44,60 @@ Skill Proficiencies:
 * Weapons - [Any]
 
 == Class Feats ==
-	----------------------
-	Battle Adrenaline [25 Nanites] -
-	Significant past training or combat experience has given you honed battle reactions, 
-	adrenaline simply aids your ability to understand and react to the task at hand. 
-	+1 non-attacking action, this turn only. Activate as a free action 
-	during your turn, limit one use per turn and a max of 3 activations per encounter.
-	|
-	|
-	V
-	Maximum of 6 activations per encounter; costs [20 Nanites] to use.
-	----------------------
-	Battle Cry / Raise Morale -
-	You roar to incite courage in your allies. 1 Action. 50% + (CHA mod) chance to 
-	raise allied effectiveness 20% (or +2 if relevant) on each of their next rolls 
-	[including your next roll]. 
-	Allies must be able to hear and understand you to get the bonus. Max 1 successful 
-	use per encounter.
-	|
-	|
-	V
-	75% + (CHA mod) success and 30% (or +3) added to next allied rolls.
-	----------------------
-	Fight Fiercely First -
-	The Soldier gets an additional +10 to initiative rolls. 
-	In addition, the soldier with this class feat gets a -3 miss chance against enemies
-	who have not taken a turn yet in combat or whom the soldier has beaten in 
-	reflex roll when room clearing until that character takes a turn after losing the 
-	reflex roll.
-	|
-	|
-	V
-	The soldier's accuracy bonus effect works on any enemy with a lower initiative until
-	that enemy has finished their 3rd turn.
-======================================
+
+Battle Hardened:
+
+    Your body and mind are used to the constant strain of combat. You no longer
+recoil quite so much from the horrors of war like a greenhorn and it takes more
+to shock you. Your pain tolerance has grown and you are more concious of your
+surroundings. You get the following:
+
+* + 8 to your Will Save
+* + 8 to your Shock Save
+* + 8 to your Reflex Save
+* + 8 to your Awareness Save
+* + 30 to your maximum health
+
+Grizzled Veteran:
+
+    You thrive on gunfighting. You love the smell of napalm in the morning. 
+You've had so much shrappnel in your body that it's gotten homeostatically used
+to it. This Feat cannot be taken unless you have "Battle Hardened" and are level 
+10. You gain the following.
+
+* An additional +20 to your maximum health
+* If you would fail a reload roll, you succeed instead    
+
+Raise Morale:
+
+	You roar to incite courage in your allies. 1 Action. 50% + (CHA mod) chance 
```

Comment:
I think this is equivalent to making a DC 50 Cha check? If so, I think it's better to phase the ability in a way that is consistent with how we usually describe making rolls.

Reply:
Confirm mathematically identical.